src/Pmi/Service/PatientStatusService.php

```diff
@@ -22,52 +22,35 @@ public function sendPatientStatusToRdr()
+                   ps.organization,
+                   ps.awardee,
+                   u.email as user_email
+            FROM patient_status_history psh
+            INNER JOIN patient_status ps ON ps.id = psh.patient_status_id
+            LEFT JOIN users u ON psh.user_id = u.id
+            WHERE psh.rdr_ts is null
+            ORDER BY psh.id ASC
+            limit {$limit}
+        ";
+        $patientStatusHistories = $this->em->fetchAll($query, []);
         $patientStatusObj = new PatientStatus($this->app);
-        foreach ($patientStatuses as $patientStatus) {
-            $query = "
-                SELECT psh.id,
-                       psh.site,
-                       psh.status,
-                       psh.comments,
-                       psh.created_ts as authored,
-                       u.email as user_email
-                FROM patient_status_history psh
-                LEFT JOIN users u ON psh.user_id = u.id
-                WHERE psh.patient_status_id = :patientStatusId
-                  AND rdr_ts is null
-                ORDER BY psh.id ASC
-            ";
-            $patientStatusHistories = $this->em->fetchAll($query, [
-                'patientStatusId' => $patientStatus['id']
-            ]);
-            foreach ($patientStatusHistories as $patientStatusHistory) {
-                $patientStatusObj->loadDataFromDb($patientStatus, $patientStatusHistory);
-                if ($patientStatusObj->sendToRdr()) {
-                    $this->em->getRepository('patient_status_history')->update($patientStatusHistory['id'], ['rdr_ts' => new \DateTime()]);
-                    $this->app->log(Log::PATIENT_STATUS_HISTORY_EDIT, [
-                        'id' => $patientStatusHistory['id']
-                    ]);
-                } else {
-                    syslog(LOG_ERR, "#{$patientStatusHistory['id']} failed sending to RDR: " . $this->rdr->getLastError());
-                }
+        foreach ($patientStatusHistories as $patientStatusHistory) {
+            $patientStatusObj->loadDataFromDb($patientStatusHistory);
+            if ($patientStatusObj->sendToRdr()) {
+                $this->em->getRepository('patient_status_history')->update($patientStatusHistory['id'], ['rdr_ts' => new \DateTime()]);
+                $this->app->log(Log::PATIENT_STATUS_HISTORY_EDIT, [
+                    'id' => $patientStatusHistory['id']
+                ]);
+            } else {
+                syslog(LOG_ERR, "#{$patientStatusHistory['id']} failed sending to RDR: " . $this->rdr->getLastError());
             }
-            // Log entry in database after all the patient status history records has been successfully sent to rdr
-            $id = $this->em->getRepository('rdr_patient_status_log')->insert([
-                    'patient_status_id' => $patientStatus['id'],
-                    'created_ts' => new \DateTime()
-                ]
-            );
-            $this->app->log(Log::RDR_PATIENT_STATUS_LOG_ADD, [
```

Comment:
Remove the `Log::RDR_PATIENT_STATUS_LOG_ADD` constant, since it's not used anymore.

Reply:
@jt2k Thanks for catching this, I also removed table `rdr_patient_status_log` from entities.